src/patternfly/components/ClipboardCopy/clipboard-copy.scss

```diff
@@ -0,0 +1,161 @@
+  --pf-c-clipboard-copy__expanded-content--PaddingLeft: var(--pf-global--spacer--md);
+  --pf-c-clipboard-copy__expanded-content--BackgroundColor: var(--pf-global--BackgroundColor--light-100);
+  --pf-c-clipboard-copy__expanded-content--BorderWidth: var(--pf-global--BorderWidth--sm);
+  --pf-c-clipboard-copy__expanded-content--BoxShadow: var(--pf-global--BoxShadow--md);
+  --pf-c-clipboard-copy__expanded-content--Top: calc(100% + var(--pf-global--spacer--xs));
+  --pf-c-clipboard-copy__expanded-content--ZIndex: var(--pf-global--ZIndex--xs);
+
+  position: relative;
+  display: inline-block;
+
+  /* stylelint-disable */
+  .pf-c-clipboard-copy__group {
+    position: relative;
+    display: flex;
+
+    .pf-c-clipboard-copy__group-toggle {
+      position: relative;
+      padding-right: var(--pf-c-clipboard-copy__group-toggle--PaddingRight);
+      padding-left: var(--pf-c-clipboard-copy__group-toggle--PaddingLeft);
+      border: none;
+
+      &::before {
+        position: absolute;
+        top: 0;
+        right: 0;
+        bottom: 0;
+        left: 0;
+        content: "";
+        border: var(--pf-c-clipboard-copy__group-toggle--BorderWidth) solid;
+        border-color: var(--pf-c-clipboard-copy__group-toggle--BorderTopColor) var(--pf-c-clipboard-copy__group-toggle--BorderRightColor) var(--pf-c-clipboard-copy__group-toggle--BorderBottomColor) var(--pf-c-clipboard-copy__group-toggle--BorderLeftColor);
+      }
+
+      &:hover,
+      .pf-m-hover {
+        &::before {
+          --pf-c-clipboard-copy__group-toggle--BorderBottomColor: var(--pf-c-clipboard-copy__group-toggle--hover--BorderBottomColor);
+        }
+      }
+
+      &:active,
+      &.pf-m-active {
+        &::before {
+          --pf-c-clipboard-copy__group-toggle--BorderBottomColor: var(--pf-c-clipboard-copy__group-toggle--active--BorderBottomColor);
+
+          border-bottom-width: var(--pf-c-clipboard-copy__group-toggle--active--BorderBottomWidth);
+        }
+      }
+    }
+
+    .pf-c-clipboard-copy__group-copy {
+      position: relative;
+      padding-right: var(--pf-c-clipboard-copy__group-copy--PaddingRight);
+      padding-left: var(--pf-c-clipboard-copy__group-copy--PaddingLeft);
+      border: none;
+
+      &::before {
+        position: absolute;
+        top: 0;
+        right: 0;
+        bottom: 0;
+        left: 0;
+        content: "";
+        border: var(--pf-c-clipboard-copy__group-copy--BorderWidth) solid;
+        border-color: var(--pf-c-clipboard-copy__group-copy--BorderTopColor) var(--pf-c-clipboard-copy__group-copy--BorderRightColor) var(--pf-c-clipboard-copy__group-copy--BorderBottomColor) var(--pf-c-clipboard-copy__group-copy--BorderLeftColor);
+      }
+
+      &:hover,
+      .pf-m-hover {
+        &::before {
+          --pf-c-clipboard-copy__group-copy--BorderBottomColor: var(--pf-c-clipboard-copy__group-copy--hover--BorderBottomColor);
+        }
+      }
```

Comment:
Looks like there are no `:active` styles for this button - is that on purpose?

Reply:
Hmm, it seems odd/inconsistent there are active styles on the expand button, but not on the copy button. If that's been confirmed with design, that's fine. Just seems a little off.

Reply:
If we're going to leave the `:active` state off, you should remove `.pf-c-clipboard-copy__group-copy` from `.pf-m-active` in the docs.

Reply:
I will confirm with design that there are no active styles as I agree it looks off.